webapp/components/expression/expressionEditorPopupState.js

```diff
@@ -18,7 +18,17 @@ const initialState = {
 }
 
 export const useExpressionEditorPopupState = (props) => {
-  const { canBeConstant, expr, mode, type, nodeDefUuidContext, nodeDefUuidCurrent, onChange, query } = props
+  const {
+    canBeConstant,
+    expr,
+    mode,
+    type,
+    nodeDefUuidContext,
+    nodeDefUuidCurrent,
+    onChange,
+    query,
+    isGroupedByParentEntity = true,
```

Comment:
can we shorten this name (and everywhere it's called this way)? groupByParent ?

Reply:
Done